src/Lecture2.hs

```diff
@@ -97,7 +108,8 @@ spaces.
 
 🕯 HINT: look into Data.Char and Prelude modules for functions you may use.
 -}
-dropSpaces = error "TODO"
+dropSpaces :: [Char] -> [Char]
+dropSpaces list =  filter (not . isSpace) list
```

Comment:
Unfortunately, this function hangs on infinite lists 😞 

If I run something like this:

```haskell
ghci> take 4 (dropSpaces ("foo" ++ repeat ' '))
```

I will observe that your implementation hangs.